modules/currencies/src/mock.rs

```diff
@@ -205,26 +203,51 @@ frame_support::construct_runtime!(
 	}
 );
 
-pub fn alice() -> AccountId {
-	<Runtime as Config>::AddressMapping::get_account_id(
-		&H160::from_str("1000000000000000000000000000000000000001").unwrap(),
-	)
+pub fn alice() -> EvmAddress {
```

Comment:
I think we can keep `alice` function to return `AccountId`, and name the new one returning `EvmAddress` as `alice_evm_addr`.